plugins/main/public/components/common/ribbon/ribbon-item.tsx

```diff
@@ -91,51 +92,53 @@ const WzRibbonItem = (props: RibbonItemProps) => {
     return `${name} ${version}`;
   };
 
-  const renderField = function <T>(field?: T): T | string {
+  const renderOptionalField = function <T>(field?: T): T | string {
     return field !== undefined || field ? field : '-';
   };
 
+  if (isOperatingSystem(item)) {
+    item.render = () => {
```

Comment:
**nitpick**: this seems to redefine a prop and this is a bad practice. Create a new object/variable to append the `render` definition.